README.md

```diff
@@ -3,88 +3,108 @@
 ![Test](https://github.com/codebuddies/django-concept/workflows/Test/badge.svg)
 [![codecov](https://codecov.io/gh/codebuddies/backend/branch/master/graph/badge.svg)](https://codecov.io/gh/codebuddies/backend)
 
+**Note:** This project is currently _in development_
 
+## Features
 
+- **Auto-reload** - modify the application code in your editor of choice. As you save changes, the application should reload automatically. There should be no need to restart containers to see code changes.
+<!-- TODO -->
 
-Background: https://github.com/codebuddies/codebuddies/issues/1136
+## Reference
 
-The API spec all the proof-of-concepts: https://app.swaggerhub.com/apis-docs/billglover/CodeBuddies/0.0.1
+- [Background]
+- [The API spec all the proof-of-concept]
```

Comment:
This should be linking to https://github.com/codebuddies/codebuddies/issues/1136 and https://app.swaggerhub.com/apis-docs/billglover/CodeBuddies/0.0.1 respectively, right? 

So:
```
The [API spec](https://app.swaggerhub.com/apis-docs/billglover/CodeBuddies/0.0.1) of our proof-of-concept

etc.
```